crates/anvil-polkadot/tests/it/demo_test.rs

```diff
@@ -0,0 +1,130 @@
+use crate::utils::TestNode;
+use anvil_core::eth::EthRequest;
+use anvil_polkadot::{cmd::NodeArgs, opts::Anvil};
+use anvil_rpc::{
+    error::{ErrorCode, RpcError},
+    response::ResponseResult,
+};
+use tokio::time::{sleep, Duration};
+
+#[tokio::test(flavor = "multi_thread")]
+async fn demo_test() {
+    let mut anvil_args = Anvil {
+        global: foundry_cli::opts::GlobalArgs::default(),
+        node: NodeArgs::default(),
+        cmd: None,
```

Comment:
we won't be needing these global params or the cmd.
Let's just pass the NodeArgs to the new_custom()